docs/api.rst

```diff
@@ -22,9 +22,9 @@ There are two main ways to query version information about the library. For guar
 
 .. data:: version_info
 
-    A named tuple that is similar to :obj:`py:sys.version_info`.
+    A named tuple that is similar to :py:obj:`sys.version_info`.
```

Comment:
This isn't entirely correct - the `py:` in the target name is used by intersphinx and corresponds to the `py` key in `intersphinx_mapping`, while the `:py:` prefix on the role just specifies the domain (not needed for `:obj:` as it only exists in the `py` domain).

The underlying issue is a hoverxref bug, as far as I can tell. It just tries all available intersphinx inventories and uses the first match, without taking the `py:` prefix in the target name into account.

For now we should probably undo this change, as this is not the only instance (e.g. `filter` [here](https://disnake--788.org.readthedocs.build/en/788/api.html#disnake.utils.find) isn't changed either).
I'll open an issue/PR for it tomorrow in the hope of getting it fixed though.

Reply:
reverted in https://github.com/DisnakeDev/disnake/pull/788/commits/0905fbbb2d184d0c2a8a9392c3084f684c414955